tutor/src/components/course-calendar/duration.cjsx

```diff
@@ -81,25 +80,17 @@ CourseDuration = React.createClass
       .value()
 
```

Comment:
Ok, I've thought about it a bit more and think we're both correctish.  I still think my method (multiply by the decimal place value) is the "correct" way to do it, but yours is fine because the # of milliseconds since the epoch won't roll over for a looong time.  Right now it's at: `1471448051077` and won't add another digit until sometime in the year 2300.  😁    So it's safe to do that for sure.

However, it did last add a digit in the year 2000.  So any dates that are before that have the potential for losing digits and sorting incorrectly.  That's still very unlikely though.

Note: I tested using: `moment().valueOf()` => `1471448219520` and then inverted it by: `moment(1471448219520).toString()`